connect-examples/v2/python_payment/requirements.txt

```diff
@@ -0,0 +1,2 @@
+squareup
+configparser
```

Comment:
This is probably a future task, but I think reading `.env` files with the `dot-env` library is now more common than reading `config.ini` with `configparser`